.travis.yml

```diff
@@ -8,23 +8,34 @@ env:
       VERSION: 7
     - DISTRIBUTION: ubuntu
       VERSION: 16.04
+#    - DISTRIBUTION: debian
+#      VERSION: 9
 
 services:
   - docker
 
 before_install:
-    # Install latest Git
+  # Install latest Git
   - sudo apt-get update
   - sudo apt-get install --only-upgrade git
-    # Allow fetching other branches than master
+  # Allow fetching other branches than master
   - git config remote.origin.fetch +refs/heads/*:refs/remotes/origin/*
-    # Fetch the branch with test code
+  # Fetch the branch with test code
   - git fetch origin docker-tests
   - git worktree add docker-tests origin/docker-tests
+  # Lint
+  - sudo pip install --upgrade pip
+  - sudo pip install ansible-lint yamllint
 
 script:
+  # Lint
+  - ansible-lint . -x ANSIBLE0016
```

Comment:
What is this ANSIBLE0016, I suppose it's an annoying warning you disable, but could you add a comment explaining which warning it is silencing ?

Reply:
it fails when we restart a service on a registered_var.changed
I'll tilt, if i see again a 'flush_handlers' in the roles